DQMOffline/L1Trigger/python/L1TMuonDQMOffline_cff.py

```diff
@@ -0,0 +1,15 @@
+import FWCore.ParameterSet.Config as cms
+
+from DQMOffline.L1Trigger.L1TMuonDQMOffline_cfi import *
+
+# modifications for the pp reference run
+muonEfficiencyThresholds_HI = [5, 7, 12]
```

Comment:
hi @thomreis - please move these lines into the file that defines l1tMuonDQMOffline

Reply:
[from line 5 to 14]

Reply:
I had that before and thought it should not be in a cfi. I will move it back.